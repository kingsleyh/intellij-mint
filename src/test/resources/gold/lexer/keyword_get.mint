module A {

  get : String {
    "Hello"
  }

}