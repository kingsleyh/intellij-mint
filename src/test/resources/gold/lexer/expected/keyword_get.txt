module ('module')
WHITE_SPACE (' ')
IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
get ('get')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('String')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
STRING_LITERAL ('"Hello"')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')